resources/views/components/meta-panel/index.blade.php

```diff
@@ -1,44 +1,60 @@
 @props([
+    'availableCheckboxFilters' => [],
+    'availableSelectFilters' => [],
     'availableSorts' => [],
-    'selectedSortOrder' => null,
-    'availableFilters' => [],
     'filterOptions' => [],
+    'selectedSortOrder' => null,
 ])
 
 <?php
-    $hasSorts = count($availableSorts) > 0;
-    $hasFilters = count($availableFilters) > 0;
+    $hasSorts = !empty($availableSorts);;
+    $hasCheckboxFilters = !empty($availableCheckboxFilters);
+    $hasSelectFilters = !empty($availableSelectFilters);
+
     if ($hasSorts) {
-        $header = $hasFilters ? "Sorts and filters" : "Sorts";
-    } else if ($hasFilters) {
+        $header = ($hasCheckboxFilters || $hasSelectFilters) ? "Sorts and filters" : "Sorts";
+    } else if ($hasCheckboxFilters) {
```

Comment:
This should also allow for only select filters.

Reply:
Fixed in latest.